chromadb/utils/embedding_functions.py

```diff
@@ -581,6 +584,51 @@ def __call__(self, input: Union[Documents, Images]) -> Embeddings:
         return embeddings
 
 
+class HuggingFaceEmbeddingServer(EmbeddingFunction[Documents]):
+    """
+    This class is used to get embeddings for a list of texts using the HuggingFace Embedding server (https://github.com/huggingface/text-embeddings-inference).
+    The embedding model is configured in the server.
+    """
+
+    def __init__(self, url: str):
+        """
+        Initialize the HuggingFaceEmbeddingServer.
+
+        Args:
+            url (str): The URL of the HuggingFace Embedding Server.
+        """
+        try:
+            import requests
+        except ImportError:
+            raise ValueError(
+                "The requests python package is not installed. Please install it with `pip install requests`"
+            )
+        self._api_url = f"{url}"
+        if not self._api_url.endswith("embed"):
+            self._api_url = f"{self._api_url}/embed".replace("//embed", "/embed")
```

Comment:
Why is this needed?

Reply:
I think the reasoning was if user provides a base URL. But I can make that clear in the docs and just remove the above.